server/src/main/java/org/apache/iotdb/db/engine/storagegroup/TsFileProcessor.java

```diff
@@ -276,7 +275,7 @@ private long getMemtableSizeThresholdBasedOnSeriesNum() {
       return config.getMemtableSizeThreshold();
     }
     long memTableSize = (long) (config.getMemtableSizeThreshold() * config.getMaxMemtableNumber()
-        / IoTDBDescriptor.getInstance().getConfig().getMemtableNumInEachStorageGroup()
+        / IoTDBDescriptor.getInstance().getConfig().getConcurrentWritingTimePartition() / 4
```

Comment:
why divide 4?

Reply:
Ok I see...
But you'd better set 4 as a constant value, and refer to the const "4" in  `addOrDeleteStorageGroup()`

Reply:
Sure~ I will fix it!